data/fields/fire_mains.json

```diff
@@ -0,0 +1,15 @@
+{
+    "key": "fire_mains",
+    "type": "combo",
+    "label": "Type",
```

Comment:
I think we need to clarify that this _Type_ is only supposed to represent the non-sprinkler part of the inlet. Maybe just calling it  _Riser Inlet_ (or _Standpipe_) would be better?!

Reply:
I think that is clear enough. `emergency=fire_service_inlet` is basically only about the inlet. So I think Riser Inlet or Standpipe would only lead to confusion.

Reply:
or maybe 'System-Type'?

Reply:
I changed it in `Network-Type`. I think this should be clear now.